src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers() {
+        this.answers = new ArrayList<>();
+    }
+
+    public Answers(List<Answer> answers) {
+        if (answers == null) {
+            this.answers = new ArrayList<>();
+            return;
```

Comment:
```suggestion
            throw new IllegalArgumentExcpetion();
```

예외 처리를 하는 것이 낫지 않을까?